tasks/help.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const ClientKitTask = require('clientkit-task');
+const Logr = require('logr');
+class HelpTask extends ClientKitTask {
+
+  constructor(name, config, runner) {
+    config.log = new Logr({
```

Comment:
I thought we might want help to have some special custom formatting options here, to make the help guide stand out better from the other tasks, but it's not necessary.